bears/general/ShebangBear.py

```diff
@@ -0,0 +1,33 @@
+import os
+from coalib.bears.LocalBear import LocalBear
+from coalib.results.Diff import Diff
+from coalib.results.Result import Result
+
+
+class ShebangBear(LocalBear):
+    AUTHORS = {'The coala developers'}
+    AUTHORS_EMAILS = {'coala-devel@googlegroups.com'}
+    LICENSE = 'AGPL-3.0'
+    CAN_FIX = {'Syntax'}
+
+    def run(self, filename, file):
+        """
+        Ensure that the file uses the generic Shebang operator.
+        """
+        if not file or file[0].startswith('#!/usr/bin/env'):
+            return
+        if file[0].startswith('#!'):
+            diff = Diff(file)
+            message = None
+            binary = os.path.basename(file[0]).strip('\n')
+            new_line = '#!/usr/bin/env {}'.format(binary)
```

Comment:
I think you are using `#!/usr/bin/env` quite often, I think it's worth to put it inside a variable as a constant :+1:

Reply:
It is only written twice here..
Putting it inside a variable would be only beneficial if we will need to change it later..

Reply:
it's also for consistency :+1:
Everything that's used more than once shouldn't be inlined ;)